core/scene.js

```diff
@@ -16,12 +16,12 @@ class Scene {
 
   loop() {
     // Calculate elapsed time
-    this.deltaTime = (this.startTime !== 0) ? now() - this.startTime : 0;
+    this.deltaTime = (this.startTime !== 0) ? Date.now() - this.startTime : 0;
```

Comment:
Ok, this is something that may sound like a very bad practice but we should have some functions in the global scope (or at least that is what experience has taught me). Let me elaborate.

Things like `Date`, `Math` and `document` are javascript reserved words, as so, compilers don't uglify them. Every time the see `Date` they leave `Date`. That means that we'll waste lots of characters in the minified version (math operations are used a lot). So, my approach was to have all of these functions in the global scope, as part of the K3J functionality. Does it make sense?

Reply:
Gotcha!